astropy/visualization/mpl_style.py

```diff
@@ -4,13 +4,6 @@
 # plotting style. It is no longer documented/recommended as of Astropy v3.0
 # but is kept here for backward-compatibility.
 
-from astropy import __minimum_matplotlib_version__
-from astropy.utils import minversion
-
-# This returns False if matplotlib cannot be imported
-MATPLOTLIB = minversion('matplotlib', __minimum_matplotlib_version__)
```

Comment:
By @saimn 's logic for `asdf`, I also don't think we need a minversion check here. @astrofrog , what do you think?